app/views/articles/_block.html.erb

```diff
@@ -1,23 +1,19 @@
 <div class="block" id="block-index-<%= i %>">
   <% if show_published %>
     <script async>
-      <%= block.published_javascript.html_safe %>
+      <%== block.published_javascript %>
```

Comment:
The change to remove all `.html_safe` in this PR make sense to me. I just wonder if there was a reason it was originally using `.html_safe`. Any idea why we might have been using `.html_safe` @rhymes?

Reply:
Nope, I don't know